Rakefile

```diff
@@ -33,6 +33,24 @@ task :default do
   system "rake --tasks"
 end
 
+desc "Generate docs for Bolt and its modules"
+task :docs do
+  require_relative 'vendored/require_vendored'
+  require "puppet-strings/tasks/generate"
+  module_paths = %w[bolt-modules/boltlib modules/aggregate modules/canary modules/puppetdb_fact].join(' ')
+  Rake::Task['strings:generate'].invoke(nil, nil, nil, nil, nil, nil, module_paths)
+end
+
+desc "Generate a JSON file containing docs for Bolt's core Puppet functions"
+task :json do
```

Comment:
This also seems to require that the `doc` directory doesn't already exist. Do we want to remove it first, or put the json in a different directory?